ts/screens/ErrorScreen.tsx

```diff
@@ -0,0 +1,79 @@
+import { Body, Button, Container, H1, Left, Text, View } from "native-base";
+import * as React from "react";
+import { Image, StyleSheet } from "react-native";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import { connect } from "react-redux";
+import AppHeader from "../components/ui/AppHeader";
+import IconFont from "../components/ui/IconFont";
+import { ReduxProps } from "../store/actions/types";
+
+type OwnProps = {
+  navigation: NavigationScreenProp<NavigationState>;
+};
+type Props = ReduxProps & OwnProps;
+
+const styles = StyleSheet.create({
+  content: {
+    alignItems: "center"
```

Comment:
have a look at `ts/components/styles/wallet.ts`, you're probably going to find some of these styles laying around